### PR TITLE
WT-10732 Set workgen keys/values for non-dynamic tables.

### PR DESCRIPTION
**Pull request by ddanderson**

### CODE REVIEW

File: bench/workgen/workgen.cpp [hunk 1]
```diff
@@ -1525,9 +1525,16 @@ ThreadRunner::op_run_setup(Operation *op)
...
 
-    // If this is not a table operation, or if it is and has a table assigned, we have
-    // nothing more to do here.
-    if (!op->is_table_op() || op->has_table()) {
+    // If this is not a table operation, we have nothing more to do here.
+    if (!op->is_table_op()) {
+        return op_run(op);
+    }
+
+    // If this is not a dynamic table operation, we still need to generate keys and values.
```
[the remark this answers is not in the document]
Comment (ddanderson) [reply bench/workgen/workgen.cpp:1533]: Thanks - I used your code and added a comment that I hope captures the intent.

**Comment by ddanderson** [comment]
@etienneptl, as Tammy is out, could you please re-review this change?

**Review by tammybailey — APPROVED** [review]
LGTM!

**Comment by ddanderson** [comment]
evergreen merge

**Pull request merged**